src/material-experimental/mdc-chips/chip-edit-input.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  Directive,
+  ElementRef,
+} from '@angular/core';
+
+/**
+ * A directive that makes a span editable and exposes functions to modify and retrieve the
+ * element's contents.
+ */
+@Directive({
+  selector: 'span[matChipEditInput]',
+  host: {
+    'class': 'mdc-chip__primary-action mat-chip-edit-input',
+    'role': 'textbox',
```

Comment:
@jelbourn are you aware of any a11y issues with this?

Reply:
I don't really know the a11y nuances of `contenteditable`, so this would require some manual testing